aptible-auth.gemspec

```diff
@@ -21,7 +21,6 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
 
   spec.add_dependency 'aptible-resource', '~> 1.0'
```

Comment:
unsure if we want to remove this, but leave it as a dep unpinned?

`spec.add_dependency 'concurrent-ruby'` example. it's verified to work for other repos with tests, so approving (as it looks like those other repos probably enforce this dep)